proto/prysm/v1alpha1/beacon_chain.proto

```diff
@@ -232,7 +239,9 @@ service BeaconChain {
     }
 
     // Server-side stream of validator information at each epoch.
+    // DEPRECATED: This endpoint is superseded by the /eth/v1/events Beacon API endpoint, there is an internal GRPC endpoint /internal/eth/v1/events
```

Comment:
`/eth/v1/events` does not stream validator info. Also same comment as for `StreamAttestations`.